com.stansassets.scene-management/Editor/Window/Tabs/SettingsTab.cs

```diff
@@ -21,6 +22,22 @@ public SettingsTab()
                 SceneManagementSettings.Instance.LandingScene = (SceneAsset)e.newValue;
                 SceneManagementSettings.Save();
             });
+            m_Enums = this.Q<VisualElement>("enums");
+            StateStackVisualizer.StackRegistered += StackRegistered;
```

Comment:
Because this part of code only subscribes to the event when stack is registered and doesn't update UI according to the current state I can: 
 1. Start playmode
 2. Open window
 3. See nothing visualized
 4. Feel disappointment

So **StateStackVisualizer** class should have:
1. Probably a single method VisualizersCollectionUpdated without arguments
2. List of currently registered visualizers
So SettingsTab can Update visualizers during playmode.